docs/contracts/v4/guides/02-manage-liquidity/04-collect.mdx

```diff
@@ -1,3 +1,84 @@
 ---
 title: Collect Fees
----
\ No newline at end of file
+---
+
+### Setup
+
+See the [setup guide](./00-setup-liquidity.mdx)
+
+# Guide
+
+In order to collect fees, the integrator must execute encoded actions
+using the `PositionManager` contract. **Note** that there is no
+`COLLECT` command, instead developers must decrease liquidity with a zero
+liquidity change.
+
+### 1. Import and define `IPositionManager`
+
+```solidity
+import {IPositionManager} from "v4-periphery/src/interfaces/IPositionManager.sol";
+// inside a contract, test, or foundry script:
+IPositionManager posm = IPositionManager(<address>);
+```
+
+### 2. Encode actions
+
+To collect fees, the following operations are require:
+* decrease - for fee collection intent
+* take pair - transfer the fees in both tokens back to owner
+
+```solidity
+import {Actions} from "v4-periphery/src/libraries/Actions.sol";
+bytes memory actions = abi.encode(Actions.DECREASE_LIQUIDITY, Actions.TAKE_PAIR);
+```
+
+### 3. Encode Parameters
+
+```solidity
+bytes[] memory params = new bytes[](2);
+```
+
+The `DECREASE_LIQUIDITY` action requires the following parameters:
+
+| Parameter    | Type      | Description                                                                   |
+|--------------|-----------|-------------------------------------------------------------------------------|
+| `tokenId`    | _uint256_ | position identifier                                                           |
+| `liquidity`  | _uint256_ | the amount of liquidity to withdraw                                           |
+| `amount0Min` | _uint128_ | the minimum amount of currency0 liquidity msg.sender is expecting to get back |
+| `amount1Min` | _uint128_ | the minimum amount of currency1 liquidity msg.sender is expecting to get back |
+| `hookData`   | _bytes_   | arbitrary data that will be forwarded to hook functions                       |
+
+**Note** that in order to collect fees we will default `liquidity`, `amount0Min` and `amount1Min` to 0.
+These parameters need to be properly set when actual liquidity needs to be withdrawn.
```

Comment:
```
**Note** because fee collection can not be manipulated in a front-run attack, it is safe to set the slippage values `amount0Min, amount1Min` to `0`
```